### DISCUSSION TITLE
Remove thread count limitation on workers

### DISCUSSION THREAD

**Discussion by brettsam** [Azure/Azure-Functions#1967] [Design discussions]
Discussion for https://github.com/Azure/Azure-Functions/issues/1962

**Comment by brettsam** [comment]
Notes
- Requires that the Python worker can determine what the host version is: https://github.com/Azure/azure-functions-host/issues/7549.
- Desirable for dynamic concurrency to be working for this, but not required.
- Some risk that customers will hit out-of-memory errors.
- But the tradeoff in default performance is desired.

**Comment by paulbatum** [comment]
We might need an out-of-memory python detector that recommends tweaking this setting

**Reply by AnatoliB** [reply]
And one for PowerShell as well

**Comment by AnatoliB** [comment]
For more details on the motivation, see [OOP Concurrency Throttling](https://microsoft.sharepoint.com/:w:/r/teams/Antares/Shared%20Documents/Functions/OOP%20Concurrency%20Throttling.docx?d=wda510823bb754173b2ec8e57286c27d7&csf=1&web=1&e=wv4HQg)
